zeppelin-zengine/pom.xml

```diff
@@ -221,6 +239,27 @@
       <scope>test</scope>
     </dependency>
 
+	<dependency>
+      <groupId>org.eclipse.jetty</groupId>
+      <artifactId>jetty-server</artifactId>
+      <version>9.2.15.v20160210</version>
+      <scope>test</scope>
+	</dependency>
+
+	<dependency>
+      <groupId>org.eclipse.jetty</groupId>
+      <artifactId>jetty-servlet</artifactId>
+      <version>9.2.15.v20160210</version>
```

Comment:
May be we could move version number to a property in a root POM, and re-use it here, below and in https://github.com/apache/incubator-zeppelin/blob/master/zeppelin-server/pom.xml#L38 what do you think?